app/src/components/validators.js

```diff
@@ -10,7 +10,7 @@ const models = {
   docGen: {
     /** @function contexts is required and must be an array */
     contexts: value => {
-      return value && Array.isArray(value);
+      return value && Array.isArray(value) && value.length > 0;
```

Comment:
An evaluation of `value.length` only is sufficient.

Reply:
changed